### DISCUSSION TITLE
Investigate bouncy particle sampler

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1546] [New methods]
See [here](https://arxiv.org/pdf/1510.02451.pdf).

**Comment by MichaelClerx** [comment]
This sounds like a party

**Comment by ben18785** [comment]
[This](https://arxiv.org/abs/1611.07873) is a good reference here.

**Comment by ben18785** [comment]
[This](https://github.com/alan-turing-institute/PDSampler.jl) software implements it so is probably good to consult.
